src/index.html

```diff
@@ -13,6 +13,38 @@
     />
```

Comment:
You are linking a SCSS file. However, browsers don't understand SCSS. You should compile your SCSS to CSS and link the CSS file instead.